modules/tfm/tfm/boards/board/RTE_Device.h

```diff
@@ -10,26 +10,30 @@
 #include <autoconf.h>
 
 /* Configuration settings for Driver_USART0. */
+#if (defined(CONFIG_TFM_SECURE_UART1) && defined(CONFIG_TFM_LOG_SHARE_UART)) \
```

Comment:
Shouldn't that be `CONFIG_TFM_SECURE_UART0` here?
Or then at lest leave a comment why UART1

Reply:
Refactored the option, that should clear it up here.